boards/arm/mimxrt1050_evk/mimxrt1050_evk.dts

```diff
@@ -19,7 +19,7 @@
 		watchdog0 = &wdog0;
 	};
 
-	chosen {
+	chosen: chosen {
```

Comment:
It needs a tag so that it can be referenced by `mimxrt1050_evk_qspi.dts` in order to add:

```
&chosen {
	zephyr,flash-controller = &flexspi;
	zephyr,flash = &is25wp064;
	zephyr,code-partition = &slot0_partition;
};
```

Reply:
> It needs a tag

https://docs.zephyrproject.org/latest/guides/dts/howtos.html#use-devicetree-overlays

It doesn't need a tag. See option 2 in the above link or just keep it the way it is.

Reply:
You are right.